staging/src/k8s.io/client-go/kubernetes/typed/core/v1/fake/fake_event_expansion.go

```diff
@@ -59,6 +73,12 @@ func (c *fakeEvents) UpdateWithEventNamespace(event *v1.Event) (*v1.Event, error
 // PatchWithEventNamespace patches an existing event. Returns the copy of the event the server returns, or an error.
 // TODO: Should take a PatchType as an argument probably.
 func (c *fakeEvents) PatchWithEventNamespace(event *v1.Event, data []byte) (*v1.Event, error) {
+	return c.PatchWithEventNamespaceWithContext(context.Background(), event, data)
```

Comment:
nit: add Deprecated comment above for consistency :)

Reply:
Fixed: https://github.com/kubernetes/kubernetes/compare/5e8be3abcf343a1912c6e4afc5b17f4ed1703445..f1834f06f4f7de8c6a5a70138cf08f8ca804f5c2